src/WeatherRequest.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from 'react';
+
+const currentWeatherURI = 'https://api.openweathermap.org/data/2.5/weather';
+const hourlyWeatherURI =
+  'https://pro.openweathermap.org/data/2.5/forecast/hourly';
+const dailyWeatherURI =
+  'https://api.openweathermap.org/data/2.5/forecast/daily';
+
+const requestLonLat = async function requestWeatherByLonAndLat(URIparams, URI) {
+  const response = await fetch(
+    `${URI}?${new URLSearchParams(URIparams).toString()}`
+  );
+  const responseJSON = await response.json();
+  return responseJSON;
+};
+
+export default function WeatherRequest(props) {
```

Comment:
Could possible break down the different weather request into their own components? Not sure how they would work in the larger scheme of things. Probably be best to discuss over Discord or something. For now I think this is okay.

Reply:
Yeah, it might be helpful to break them down.